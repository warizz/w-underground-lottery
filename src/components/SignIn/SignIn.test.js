import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Uut from './';

describe('snapshot testing', () => {
  it('should match snapshot when no props', () => {
    const wrapper = shallow(<Uut />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });

  it('should match snapshot with props.errorText', () => {
    const props = {
      errorText: 'errorText',
    };
    const wrapper = shallow(<Uut {...props} />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
