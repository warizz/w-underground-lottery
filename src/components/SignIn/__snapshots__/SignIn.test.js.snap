// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing should match snapshot when no props 1`] = `
<div
  className="SignIn"
>
  <Button
    as="button"
    loading={false}
    onClick={[Function]}
    primary={true}
  >
    Sign in with Facebook
  </Button>
  <Message
    info={true}
  >
    <MessageHeader>
      Why log in with facebook?
    </MessageHeader>
    <p>
      - Don\\'t reinvent the wheel: facebook already have great security by 1000 top class engineers keeping your password safe.
    </p>
    <p>
      - This app need only your username and profile picture, it can do no harm.
    </p>
  </Message>
</div>
`;

exports[`snapshot testing should match snapshot with props.errorText 1`] = `
<div
  className="SignIn"
>
  <Button
    as="button"
    loading={false}
    onClick={[Function]}
    primary={true}
  >
    Sign in with Facebook
  </Button>
  <Message
    info={true}
  >
    <MessageHeader>
      Why log in with facebook?
    </MessageHeader>
    <p>
      - Don\\'t reinvent the wheel: facebook already have great security by 1000 top class engineers keeping your password safe.
    </p>
    <p>
      - This app need only your username and profile picture, it can do no harm.
    </p>
  </Message>
  <Message
    negative={true}
  >
    <MessageHeader>
      An error occurred
    </MessageHeader>
    <p>
      errorText
    </p>
  </Message>
</div>
`;
